src/operator/quantization/mkldnn/mkldnn_quantized_pooling.cc

```diff
@@ -39,7 +39,7 @@ static void MKLDNNQuantizedPoolingForward(const nnvm::NodeAttrs& attrs,
       << "mkldnn_quantized_pooling op only supports uint8 and int8 as input "
          "type";
   const PoolingParam& param = nnvm::get<PoolingParam>(attrs.parsed);
-  MKLDNNPoolingCompute(ctx, param, in_data[0], req[0], out_data[0], nullptr);
+  MKLDNNPoolingCompute(ctx, param, in_data[0], req[0], out_data[0], nullptr, false);
```

Comment:
Could you please add a comment before false?
```suggestion
  MKLDNNPoolingCompute(ctx, param, in_data[0], req[0], out_data[0], nullptr, /*use_adaptive*/ false);
```